src/Kentico.Xperience.Lucene/Admin/Components/LuceneIndexConfigurationComponent.cs

```diff
@@ -17,26 +17,26 @@ public class LuceneIndexConfigurationComponentProperties : FormComponentProperti
 }
 #pragma warning restore
 
-public class LuceneIndexConfigurationComponentClientProperties : FormComponentClientProperties<List<LuceneIndexIncludedPath>>
+public class LuceneIndexConfigurationComponentClientProperties : FormComponentClientProperties<IEnumerable<LuceneIndexIncludedPath>>
 {
-    public List<string>? PossibleItems { get; set; }
+    public IEnumerable<string>? PossibleItems { get; set; }
 }
 
 public sealed class LuceneIndexConfigurationComponentAttribute : FormComponentAttribute
 {
 }
 
 [ComponentAttribute(typeof(LuceneIndexConfigurationComponentAttribute))]
-public class LuceneIndexConfigurationComponent : FormComponent<LuceneIndexConfigurationComponentProperties, LuceneIndexConfigurationComponentClientProperties, List<LuceneIndexIncludedPath>>
+public class LuceneIndexConfigurationComponent : FormComponent<LuceneIndexConfigurationComponentProperties, LuceneIndexConfigurationComponentClientProperties, IEnumerable<LuceneIndexIncludedPath>>
```

Comment:
Changed from `List` to `IEnumerable` because Xperience's built in collection based Form Components use `IEnumerable`